cli/cli/commands/enclave/dump/dump.go

```diff
@@ -85,16 +91,51 @@ func run(
 	}
 
 	enclaveUuid := enclaveInfo.GetEnclaveUuid()
-
 	if enclaveOutputDirpath == defaultEnclaveDumpDir {
 		enclaveName := enclaveInfo.GetName()
 		enclaveOutputDirpath = fmt.Sprintf("%s%s%s", enclaveName, enclaveDumpSeparator, enclaveUuid)
 	}
 
-	if err := kurtosisBackend.DumpEnclave(ctx, enclave.EnclaveUUID(enclaveUuid), enclaveOutputDirpath); err != nil {
+	if err = kurtosisBackend.DumpEnclave(ctx, enclave.EnclaveUUID(enclaveUuid), enclaveOutputDirpath); err != nil {
 		return stacktrace.Propagate(err, "An error occurred dumping enclave '%v' to '%v'", enclaveIdentifier, enclaveOutputDirpath)
 	}
 
+	if enclaveInfo.ApiContainerStatus != kurtosis_engine_rpc_api_bindings.EnclaveAPIContainerStatus_EnclaveAPIContainerStatus_RUNNING {
+		logrus.Debugf("Couldn't dump file information as the enclave '%v' is not running", enclaveIdentifier)
```

Comment:
I think some users could miss this message if they set only Info level, Do you think could be worth it to change to info?

Reply:
Good shout!